src/EnergyPlus/PlantCondLoopOperation.cc

```diff
@@ -1245,10 +1245,9 @@ void LoadEquipList(EnergyPlusData &state,
             FoundIntendedList = true;
             // get object item for real this time
             {
-                auto const SELECT_CASE_var(state.dataPlantCondLoopOp->EquipListsTypeList(Num));
-                if (SELECT_CASE_var == LoopType::Plant) {
+                if (state.dataPlantCondLoopOp->EquipListsTypeList(Num) == LoopType::Plant) {
                     CurrentModuleObject = "PlantEquipmentList";
-                } else if (SELECT_CASE_var == LoopType::Condenser) {
+                } else if (state.dataPlantCondLoopOp->EquipListsTypeList(Num) == LoopType::Condenser) {
```

Comment:
One thing about switch blocks is that the IDEs will force you to justify if you don't handle every possible enum choice.  With IF blocks you can easily slip enum cases by without handling them.  (Of course with switch blocks it is easy to forget a `break`)